### PR TITLE
fix(apig/validate): update validate message

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
Some validate messages are wrong.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. update validate messages.
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: huaweicloud/services/apig/resource_huaweicloud_apig_api.go [hunk 1]
```diff
@@ -90,8 +90,8 @@ func ResourceApigAPIV2() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 				ValidateFunc: validation.StringMatch(regexp.MustCompile("^([\u4e00-\u9fa5A-Za-z][\u4e00-\u9fa5A-Za-z_0-9]{2,63})$"),
-					"The name contains of 3 to 64 characters, starting with a letter. Only letters, digits, "+
-						"hyphens (-) and underscore (_) are allowed."),
+					"The name contain of 3 to 64 characters and only letters, digits, underscore (_) and chinese "+
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/apig/resource_huaweicloud_apig_api.go]: s/contain of/contains or use consists of

File: huaweicloud/services/apig/resource_huaweicloud_apig_throttling_policy.go [hunk 2]
```diff
@@ -62,8 +62,8 @@ func ResourceApigThrottlingPolicyV2() *schema.Resource {
 				Required: true,
 				ValidateFunc: validation.StringMatch(
 					regexp.MustCompile("^([\u4e00-\u9fa5A-Za-z][\u4e00-\u9fa5A-Za-z_0-9]{2,63})$"),
-					"The name contains of 3 to 64 characters, starting with a letter. Only letters, digits "+
-						"and underscore (_) are allowed."),
+					"The name contain of 3 to 64 characters and only letters, digits, underscore (_) and chinese "+
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/apig/resource_huaweicloud_apig_throttling_policy.go]: ditto

File: huaweicloud/services/apig/resource_huaweicloud_apig_vpc_channel.go [hunk 3]
```diff
@@ -65,8 +65,8 @@ func ResourceApigVpcChannelV2() *schema.Resource {
 				Required: true,
 				ValidateFunc: validation.StringMatch(
 					regexp.MustCompile("^([\u4e00-\u9fa5A-Za-z][\u4e00-\u9fa5A-Za-z-_0-9]{2,63})$"),
-					"The name contains of 3 to 64 characters, starting with a letter. Only letters, digits, "+
-						"hyphens (-) and underscore (_) are allowed."),
+					"The name contain of 3 to 64 characters and only letters, digits, underscore (_), hyphens (-) "+
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/apig/resource_huaweicloud_apig_vpc_channel.go]: ditto

File: huaweicloud/services/apig/resource_huaweicloud_apig_throttling_policy.go (hunk 2 above)
Comment (Lance52259) [reply huaweicloud/services/apig/resource_huaweicloud_apig_throttling_policy.go]: done.

File: huaweicloud/services/apig/resource_huaweicloud_apig_api.go (hunk 1 above)
Comment (Lance52259) [reply huaweicloud/services/apig/resource_huaweicloud_apig_api.go]: done.

File: huaweicloud/services/apig/resource_huaweicloud_apig_vpc_channel.go (hunk 3 above)
Comment (Lance52259) [reply huaweicloud/services/apig/resource_huaweicloud_apig_vpc_channel.go]: done.

**Pull request merged**